pmlib/Resonator1D.py

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
```suggestion
```

Reply:
Ok. This was a convention that I use when there is no comment at the top, maybe should avoid it from now on.